public/css/main.css

```diff
@@ -20,14 +19,96 @@ a {
 }
 
 /*** GENERAL ***/
-
 body {
   background-color: #F2F2F2;
 }
 
 /*** HEADER BAR ***/
+.header {
+  background-color: #409A93;
+  font-family: 'Noto Sans', sans-serif;
+  padding: 2.5rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.naz-events{
+  color: white;
+  font-weight: bold;
+  font-size: 2.3rem;
+}
+
+.calendar {
+  font-size: 2.3rem;
+  color: white;
+  margin-top: 0.1rem;
+}
+
+.plus{
+  margin-top: 0.2rem;
+  font-size: 2.3rem;
+  color: white;
+}
+
+.languageNav {
+  background: #F2F2F2;
+  width: 100%;
+  padding: 0.6rem;
+  box-shadow: 0.4rem 0.5rem 0.5rem #DBDBDB;
+}
+
+.langBtn{
+  width: 48.5%;
```

Comment:
The sizing of these could be made easier - set `.languageNav` to `display: flex` and give these buttons `flex-grow: 1`, then just put a `margin-right` on the left button, equal to the padding of `.languageNav`

Reply:
What you have is close, but as you resize the page the gap between the two buttons is noticeable - the CSS should be rock solid so that that gap never changes